lib/ui/text.dart

```diff
@@ -223,7 +223,7 @@ enum TextDecorationStyle {
 /// Returns true if the lists are both null, or if they are both non-null, have
 /// the same length, and contain the same elements in the same order. Returns
 /// false otherwise.
-bool _listEquals<T>(List<T> a, List<T> b) {
+bool listEquals<T>(List<T> a, List<T> b) {
```

Comment:
I don't think we should make this public.  It will likely fail on cases where people would expect it not to (like, for example, comparing lists of `Map`s).

We should still be able to use it from painting.dart without renaming it here - because this is a part library, private stuff is available all over.